src/app/components/Byline/index.test.tsx

```diff
@@ -54,25 +58,28 @@ describe('Byline', () => {
   it('should render a list when required data is passed correctly', () => {
     render(<Byline blocks={bylineWithNameAndRole} />);
 
-    const list = screen.getByRole('list');
+    const list = screen.getAllByRole('list');
 
-    expect(list).toBeInTheDocument();
+    expect(list[0]).toBeInTheDocument();
```

Comment:
Just an FYI, you can use array destrucuring here

```suggestion
    const [list] = screen.getAllByRole('list');

    expect(list).toBeInTheDocument();
```